docs/js/modal.md

```diff
@@ -49,11 +46,12 @@ Based on this markup, the component will get all content inside of `[data-modal]
 
 ## Options
 
-Modal provides three customizable options: `container`, `triggerClose` and `triggerOpen`. By default `.modal` is appended in the `body`, `triggerClose` and  `triggerOpen` are `null`.
+Modal provides some customizable options as: `container`, `size`, `triggerClose` and `triggerOpen`. The default of `.modal` is to be appended in the `body`, the `size` is 'medium', `triggerClose` and  `triggerOpen` are `null`.
```

Comment:
`such as` ao invés de `as`
Elabora um pouco mais nessa segunda oração. Acho que faltam algumas palavras que poderiam deixar mais claro, tipo `The default ?? for the .modal ?? is to be ... `. Dae na segunda parte acho que seria melhor deixar em uma oração separada e fazer ajuste tbm seguindo o mesmo citado acima.